src/Processors/Formats/Impl/ORCBlockOutputFormat.cpp

```diff
@@ -28,6 +28,34 @@ namespace DB
 namespace ErrorCodes
 {
     extern const int ILLEGAL_COLUMN;
+    extern const int NOT_IMPLEMENTED;
+}
+
+namespace
+{
+
+orc::CompressionKind getORCCompression(FormatSettings::ORCCompression method)
+{
+    if (method == FormatSettings::ORCCompression::NONE)
```

Comment:
why not a switch case?

Reply:
Because of `ifdef`. With switch we cannt build clickhouse: 
```
/build/src/Processors/Formats/Impl/ParquetBlockOutputFormat.cpp:53:29: error: code will never be executed [-Werror,-Wunreachable-code]
Mar 06 16:32:26             throw Exception(ErrorCodes::NOT_IMPLEMENTED, "Brotli compression method is not supported");
```